### DISCUSSION TITLE
Allow to change the keyboard shortcut to comment/uncomment code

### DISCUSSION THREAD

**Discussion by SergioSuarezGil** [Kong/insomnia#4112] [Ideas]
Shortcut to comment/uncomment in JSON & GraphQL

<img width="500" alt="Captura de pantalla 2021-10-14 a las 9 46 27" src="https://user-images.githubusercontent.com/3636232/137273942-a08763f0-5687-4e72-9172-12fe03f7a743.png">

`CMD+/` doesn't work on European keyboards because it would be `CMD+SHIFT+/` so it would be nice to be able to configure that shortcut.

![](https://user-images.githubusercontent.com/3636232/137198350-bb0682df-98dc-42a1-8cd6-374947d43486.png)

<img width="887" alt="Captura de pantalla 2021-10-14 a las 9 41 30" src="https://user-images.githubusercontent.com/3636232/137274339-4e30fd7c-4a56-4bfc-8c11-4bec345c7df1.png">

**Comment by SergioSuarezGil** [comment]
Based on issue https://github.com/Kong/insomnia/pull/2649

> https://github.com/Kong/insomnia/blob/cbc1cfc8b8e86822ec5303ff1f8310e2795c8258/packages/insomnia-app/app/ui/components/codemirror/code-editor.tsx#L78

**Comment by iamkhav** [comment]
Hey, I know this is a bit more than two years late but here we go: https://github.com/Kong/insomnia/pull/6889
